src/svg-icon.jsx

```diff
@@ -112,12 +105,17 @@ const SvgIcon = React.createClass({
       ...other,
     } = this.props;
 
+    const {
+      baseTheme,
+      prepareStyles,
+    } = this.state.muiTheme;
+
     const offColor = color ? color :
       style && style.fill ? style.fill :
-      this.state.muiTheme.rawTheme.palette.textColor;
+      baseTheme.palette.textColor;
     const onColor = hoverColor ? hoverColor : offColor;
 
-    const mergedStyles = this.mergeStyles({
+    const mergedStyles = Object.assign({
```

Comment:
let's have @oliviertassinari take a look at this too. it might be a very very silent and rare breaking change. in a normal day I'd say  let's keep the migration safe. I'm not seeing anything particular about this day though :laughing: :laughing: